native/cocos/network/DownloaderImpl-apple.mm

```diff
@@ -656,7 +663,12 @@ - (void)URLSession:(NSURLSession *)session downloadTask:(NSURLSessionDownloadTas
         }
         ccstd::vector<unsigned char> buf; // just a placeholder
         _outer->onTaskFinish(*[wrapper get], errorCode, errorCodeInternal, errorString, buf);
-        self.hasUnfinishedTask = NO;
+
+        NSMutableDictionary *dict = [NSMutableDictionary dictionary];
+        NSString *requesetURL = [NSString stringWithFormat:@"%s", [wrapper get]->requestURL.c_str()];
+        [dict removeObjectForKey:requesetURL];
+        NSString *savaPath = [NSString stringWithFormat:@"%s", (cc::FileUtils::getInstance()->getWritablePath() + "resumeData.plist").c_str()];
+        [dict writeToFile:savaPath atomically:YES];
```

Comment:
Duplicate codes. Can implement it in a function?

Reply:
Slightly duplicate but with different purposes, I don't think it is necessary to reuse